dietpi/dietpi-software

```diff
@@ -6427,6 +6435,53 @@ _EOF_
 			Download_Test_Media
 		fi
 
+		if To_Install 212 kavita # Kavita
+		then
+			case $G_HW_ARCH in
+				1|2) local arch='arm';;
+				3) local arch='arm64';;
+				*) local arch='x64';;
+			esac
+
+			local fallback_url="https://github.com/Kareadita/Kavita/releases/download/v0.7.8/kavita-linux-$arch.tar.gz"
+			Download_Install "$(curl -sSfL 'https://api.github.com/repos/Kareadita/Kavita/releases/latest' | mawk -F\" "/\"browser_download_url\": .*\/kavita-linux-$arch\.tar\.gz\"/{print \$4}")" /opt
+
+			# User
+			Create_User -g dietpi -d /mnt/dietpi_userdata/kavita kavita
```

Comment:
As we do not create/use `/mnt/dietpi_userdata/kavita`, make the install dir its home dir:
```suggestion
			Create_User -g dietpi -d /opt/kavita kavita
```